proc_scripts/load_data.py

```diff
@@ -210,17 +398,56 @@ def proc_DOSY_CPMG(s):
     fl.image(abs(s))
     s.chunk('t2',['echo','t2'],[l25,-1])
     s.reorder(['m','n','ph4','ph8','indirect','echo','t2'])
-    s.ft('t2', shift=True).ift('t2') # this is overkill -- need a pyspecdata function that does this w/out the fft
-    # }}}
+    s.ft('t2', shift=True)
     return s
 
-postproc_dict = {'ag_IR2H':proc_bruker_deut_IR_withecho_mancyc,
-        'ab_ir2h':proc_bruker_deut_IR_mancyc,
+def proc_ESR_linewidth(s):
+    logger.info(strm("pre-processing for ESR linewidth calculation"))
+    s -= s['$B_0$',:50].C.mean('$B_0$')
+    s_integral = s.C.run_nopop(cumsum,'$B_0$')
+    x1,x2 = s_integral.getaxis('$B_0$')[r_[5,-5]]
+    y1 = s_integral.data[:5].mean()        """calculate the fit function along the axis taxis.
+        
+        Parameters
+        ----------
+        taxis: ndarray, int
+            :if ndarray: the new axis coordinates along which we want to calculate the fit.
+            :if int: number of evenly spaced points along the t-axis along the fit
+        set_what: 'str', optional
+            forcibly sets a specific symbol
+        set_to: double, optional
+            the specific value (int) you are assigning the symbol you included
+
+        Returns
+        -------
+        self: nddata
+            the fit function evaluated along the axis coordinates that were passed
+        """
+
```

Comment:
what is going on here??

Reply:
lol I saw this before this comment and thought the same thing! I deleted sorry about that! I don't know how I didn't catch this!